### PR TITLE
Feature/display settings

### PR DESCRIPTION
**Pull request by ghislaineguerin**

### CODE REVIEW

File: gulpfile.js [hunk 1]
```diff
@@ -264,7 +264,7 @@ gulp.task("test:e2e", function (cb) {
 
 gulp.task("metrics", factory.metrics());
 gulp.task("test",  function (cb) {
-  runSequence(["config", "html2js"], ["test:unit", "test:e2e"], "coveralls", cb);
```
Comment (settinghead) [inline gulpfile.js]: @ghislaineguerin are we getting rid of e2e tests in CI?

File: web/partials/displays/proto-screenshot-states.html [hunk 2]
```diff
@@ -0,0 +1,63 @@
+<div class="l_display-screenshot">
+  <div class="u_lh-normal half-bottom">
```
Comment (alex-deaconu) [inline web/partials/displays/proto-screenshot-states.html]: @ghislaineguerin curious, are these standard classes? Thanks.

File: web/partials/displays/display-fields.html [hunk 3]
```diff
@@ -1,247 +1,223 @@
...
+            </span> 
+          </td>
+          <td class="table-body__cell hidden-xs proto-keep"><span>1-Jun-2016 3:05 PM</span></td>
+        </tr>
+      </tbody>
+    </table>
   </div>
+</div>
 
+<div class="add-top u_lh-normal">
```
Comment (alex-deaconu) [inline web/partials/displays/display-fields.html]: @ghislaineguerin is this a standard class? Thanks.

File: web/partials/displays/display-fields.html [hunk 4]
```diff
@@ -1,247 +1,223 @@
...
+          <td class="table-body__cell status proto-keep" style="display:none;">
+            <span class="text-danger"><i class="fa fa-times"></i> Offline</span> 
+          </td>
+          <td class="table-body__cell status proto-keep" style="display:none;">
+            <!-- <img src="https://s3.amazonaws.com/Rise-Images/Icons/offline_b.svg" alt="Rise Player Not Installed" title="Rise Player Not Installed"> -->
+            <span class="status-not-installed add-right"> 
+              <button class="btn btn-xs btn-primary">Install Rise Player</button>
+            </span> 
+          </td>
+          <td class="table-body__cell hidden-xs proto-keep"><span>1-Jun-2016 3:05 PM</span></td>
```
Comment (alex-deaconu) [inline web/partials/displays/display-fields.html]: @ghislaineguerin seeing a lot of `proto-keep` classes in this block. Wondering if that's standard. Thanks.

File: gulpfile.js (hunk 1 above)
Comment (ghislaineguerin) [inline gulpfile.js]: @settinghead we do this for prototypes as they tend to break the builds. This is not an actual change.

File: web/partials/displays/proto-screenshot-states.html (hunk 2 above)
Comment (ghislaineguerin) [inline web/partials/displays/proto-screenshot-states.html]: @alex-deaconu yes, these are layout (l_) and utility (u_) classes. Part of the SMACSS methodology changes.

Comment (alex-deaconu) [inline web/partials/displays/proto-screenshot-states.html]: @ghislaineguerin 👍 . Thanks for the clarification.

File: gulpfile.js (hunk 1 above)
Comment (settinghead) [inline gulpfile.js]: @ghislaineguerin thanks

File: web/partials/displays/display-fields.html (hunk 3 above)
Comment (ghislaineguerin) [inline web/partials/displays/display-fields.html]: @alex-deaconu I've removed the proto-keep classes.

Comment (alex-deaconu) [inline web/partials/displays/display-fields.html]: @ghislaineguerin awesome, thanks!

**Pull request closed without merging**